src/headers/gpm.h

```diff
@@ -25,6 +25,18 @@
 
 #include <paths.h>              /* _PATH_VARRUN etc. */
 
+#ifdef HAVE_NCURSES_H
```

Comment:
this is a public header to be included by other projects. All of those `HAVE_*` macros may be undefined. In that case, the later use of `WINDOW` will fail to compile.